sdk/python/v1beta1/setup.py

```diff
@@ -37,6 +38,19 @@
         "kubeflow/katib/katib_api_pb2.py",
     )
 
+if os.path.exists(katib_grpc_svc_file):
+    shutil.copy(
+        katib_grpc_svc_file,
+        "kubeflow/katib/katib_api_pb2_grpc.py",
+    )
+
+    with open("kubeflow/katib/katib_api_pb2_grpc.py", 'r+') as file:
+        content = file.read()
+        new_content = content.replace("api_pb2", "kubeflow.katib.katib_api_pb2")
```

Comment:
Is there a way to avoid this tricky import change ?
Maybe we could add those protobuf files as part of SDK `__init__.py` ?
https://github.com/kubeflow/katib/blob/8b095add55d7dd8983eba3d3b4bab34d51dc2f35/sdk/python/v1beta1/kubeflow/katib/__init__.py

Reply:
Yeah, we can achieve this by not renaming those files, i.e. copy `api_pb2.py` to python SDK as `api_pb2.py` rather than `katib_api_pb2.py`.

Reply:
@andreyvelich I think we may need to do this tricky import change, because adding `import kubeflow.katib.api_pb2 as api_pb2` to `__init__.py` makes no difference in resolving the import error:

1. `__init__.py` and `api_pb2_grpc.py` are different modules in this directory. Importing `api_pb2` in `__init__.py` will not affect package import in `api_pb2_grpc.py`.
2. Python3 has banned relative package import like `import api_pb2`. Python will search for `api_pb2.py` in the `sys.path` rather than the same directory. If we want to use relative import, we need to specify the package path like `from . import <filename>`.

Reply:
I see that makes sense, @droctothorpe @tenzen-y @johnugeorge any thoughts ?
I guess, we still need to modify the import to make sure we import the `api_pb2.py`  from `kubeflow.katib` package.

Reply:
@Electronic-Waste This import error happened only in the CI, and you have not seen it in your local, right?

Reply:
@tenzen-y In fact, the import error happens in local. It occurred in both pytest (`make pytest`) and SDK importing (`python setup.py install --user` and uses it in the demo)

Reply:
I see.
I'm wondering if we can specify the Python API client package name here: https://github.com/kubeflow/katib/blob/4a385f515afd0985e843bb5d31e0b3a6c2e17b8b/pkg/apis/manager/v1beta1/buf.gen.yaml#L17-L18

If it is possible, I'm thinking that we can avoid this hack approach.

@Electronic-Waste Could you investigate it?

Reply:
Sure，I'll investigate it.

Reply:
@tenzen-y I've found a related issue describing this: https://github.com/protocolbuffers/protobuf/issues/7061

The `python_package` option is still under developing now. So I think we can't avoid this hack approach now :)

Reply:
@Electronic-Waste Please can you add the TODO comment in your post-gen script to track this issue, so maybe in the future we can remove this import re-write.

Reply:
@andreyvelich I've added the TODO comment to the post-gen scripts now :)

Reply:
Thanks for investigating this problem.